packages/nameguard-sdk/src/index.ts

```diff
@@ -397,9 +397,24 @@ export interface NameGuardReport extends ConsolidatedNameGuardReport {
    * `canonical_name` is guaranteed to be normalized with the exception of the case
    * where `normalization` is `unknown` and one or more labels are represented as `[labelhash]`.
    */
-  canonical_name: string | null;
+  canonical_name?: string;
+}
+
+export interface InspectedNameGuardReport extends AbstractNameGuardReport {
+    labels: LabelGuardReport[];
```

Comment:
Please see my other comment suggesting adding a definition here for the following field:

`canonical_name: string;`

Reply:
It still can be undefined in InspectedNameGuardReport.

Reply:
It is documented in AbstractNameGuard:
```ts
  /**
   * The name considered to be the canonical form of the analyzed `name`.
   *
   * `undefined` if and only if the canonical form of `name` is considered to be undefined or `name` was uninspected and this is an ``UninspectedNameGuardReport`.
   *
   * If a label is represented as `[labelhash]` in `name`,
   * the `canonical_name` will also contain the label represented as `[labelhash]`.
   *
   * `canonical_name` is guaranteed to be normalized with the exception of the case
   * where `normalization` is `unknown` and one or more labels are represented as `[labelhash]`.
   */
  canonical_name?: string;
```